mealpy.py

```diff
@@ -23,11 +25,20 @@
 }
 
 
+def load_config():
+    schema = strictyaml.Map({
+        'email_address': strictyaml.Str(),
```

Comment:
Looks like there is an [email specific version](https://hitchdev.com/strictyaml/using/alpha/scalar/email-and-url/) for this, `Email()`?